interface/orders/types_edit.php

```diff
@@ -162,247 +232,363 @@ function proc_type_changed() {
+                                a hierarchical manner");?>.</p>
+                            <p><?php echo xlt("Group - First tier - For a group of tests e.g. Organ/Disease Panel");?></p>
+                            <p><?php echo xlt("Procedure Type - Second Tier - For a Test containing a collection of tests e.g. Electrolyte Panel");?></p>
+                            <p><?php echo xlt("Discrete Result - Third Tier - The actual test names to hold the results returned Na, K, Cl, CO2 and Default Units, Default Range etc");?></p>
+                            <p><?php echo xlt("Recommendation - Optional");?></p>
+                            <p><?php echo xlt("For single tests: Group >Procedure Type > Discrete results");?></p>
+                            <p><?php echo xlt("Group - Individual Tests");?></p>
+                            <p><?php echo xlt("Procedure Order - Serum Uric Acid");?></p>
+                            <p><?php echo xlt("Discrete Result - Serum Uric Acid - will hold the returned result value");?></p>
+                            <p><?php echo xlt("The difference between the two is that for a panel of tests that are ordered together the individual tests are represented by Discrete Result only and these tests cannot be ordered separately unless they have also been setup as single tests");?></p>
+                            <p><?php echo xlt("As the fist step choose Group as the Top Level Tier 1 and fill in the required details");?></p>
+                            <p><?php echo xlt("For detailed instructions close the 'Enter Details' popup and click on the Help icon on the main form. ");?><i class="fa fa-question-circle" aria-hidden="true"></i></p>
+                        </div>
+                    </div>
+                    <div class="col-xs-12">
+                        <div class="clearfix">
+                            <div class="col-sm-12 label-div">
+                                <label class="control-label" for="form_name"><?php echo xlt('Name'); ?>:</label><a href="#name_info" class="icon-tooltip" data-toggle="collapse"><i class="fa fa-question-circle" aria-hidden="true"></i></a>
+                            </div>
+                            <div class="col-sm-12">
+                                <input type='text' name='form_name' id='form_name 'maxlength='63'
+                                    value='<?php echo htmlspecialchars($row['name'], ENT_QUOTES); ?>'
+                                    title='<?php echo xlt('Your name for this category, procedure or result'); ?>'
+                                     class='form-control'>
+                            </div>
+                        </div>
+                        <div id="name_info" class="col-sm-12 collapse">
+                            <a href="#name_info" data-toggle="collapse" class="pull-right"><i class="fa fa-times" style="color:gray" aria-hidden="true"></i></a>
```

Comment:
I am seeing a lot of "pull-right". Is this something that caused an issue in RTL stuff and why we have the oe-text-to-right and oe-text-to-left or is this not relate?

Reply:
Sorry about the pull-rights, I had finalized this feature a long time ago before the oe-text-to-right issue was highlighted. I was under the impression that it has not yet been implemented. I would appreciate if you could summarize it here :)